app/views/posts/_images_display.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
```

Comment:
can you change the file name to image_carousel or something like that? just to make it more relevant